### PR TITLE
Auth testing session

### PR DESCRIPTION
**Pull request by flypaper0**
# Description

Resolves # (issue)

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

**Review by llbartekll — APPROVED** [review]

File: Sources/Auth/AuthClientFactory.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@ import WalletConnectNetworking
 public struct AuthClientFactory {
 
     public static func create(metadata: AppMetadata, account: Account?, relayClient: RelayClient) -> AuthClient {
-        let logger = ConsoleLogger(loggingLevel: .off)
+        let logger = ConsoleLogger(loggingLevel: .debug)
```
Comment (flypaper0) [inline Sources/Auth/AuthClientFactory.swift:11]: .off

File: Sources/WalletConnectNetworking/NetworkInteractor.swift [hunk 2]
```diff
@@ -60,7 +60,7 @@ public class NetworkingInteractor: NetworkInteracting {
         return requestPublisher
             .filter { $0.request.method == request.method }
             .compactMap { topic, rpcRequest in
-                guard let id = rpcRequest.id, let request = try? rpcRequest.params?.get(Request.self) else { return nil }
+                guard let id = rpcRequest.id, let request = try! rpcRequest.params?.get(Request.self) else { return nil }
```
Comment (flypaper0) [inline Sources/WalletConnectNetworking/NetworkInteractor.swift:63]: !

File: Sources/Auth/Services/Wallet/WalletRequestSubscriber.swift [hunk 3]
```diff
@@ -33,20 +36,14 @@ class WalletRequestSubscriber {
         networkingInteractor.requestSubscription(on: AuthProtocolMethod.authRequest)
             .sink { [unowned self] (payload: RequestSubscriptionPayload<AuthRequestParams>) in
                 logger.debug("WalletRequestSubscriber: Received request")
-                let message = messageFormatter.formatMessage(from: payload.request.payloadParams, address: address)
+                guard let message = messageFormatter.formatMessage(from: payload.request.payloadParams, address: address) else {
+                    Task {
```
Comment (flypaper0) [inline Sources/Auth/Services/Wallet/WalletRequestSubscriber.swift:40]: priority hight?

Comment (flypaper0) [inline Sources/Auth/Services/Wallet/WalletRequestSubscriber.swift]: priority hight?

**Pull request merged**